testpmd/hooks/user-tests.yml

```diff
@@ -3,3 +3,21 @@
   include_role:
     name: example-cnf-validate
   when: run_migration_test|default(true)|bool
+
+- name: Get logs from pods
+  community.kubernetes.k8s_log:
```

Comment:
In the next release, I am planning to have multiple containers in the testpmd and trex pods. So it will be required to pass "container" name. We may need to add this support later, if this cannot be generalized now.